ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml

```diff
@@ -1057,7 +1057,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )nfd-images,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
I don't think we need `always_run: true` on `ci-artifacts` PR, rigth?
(there's a way to "always run if some files are modified, Ecosystem team did it in their fork, but I didn't try it yet)

Reply:
done

Reply:
You can take a look here https://github.com/openshift/release/blob/2849dfe791a13176875264a23928397b644204ab/ci-operator/config/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master.yaml#L34